myuw/dao/student.py

```diff
@@ -0,0 +1,129 @@
+"""
+This module will provide a
+"""
+from uw_sws.enrollment import enrollment_search_by_regid
```

Comment:
Please change to use myuw.dao.enrollment: enrollment_search, which is prefetched in the request